resources/views/account/accept-eula.blade.php

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML 5 Boilerplate</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+
+    @if ($signature)
+        <center>
+            <img src="{{ $logo }}">
+        </center>
+    @endif
+    <br>
+    
+    <h2>{{ $item->name }}</h2>
+    <p>Date: {{ date('Y-m-d h:i') }} </p>
```

Comment:
We may want to use the helper here to format the date the way the user has specified in the settings.

Reply:
We should also figure out what we want this page to look like. What info it should have, etc.

And probably the option to email the agreeing user a copy of the PDF as an attachment. That's not as hard as it seems (at least not last time I tried it.) You can see how we did that in the InvoiceChargeSucceeded.php file in the `app/Notifications` dir in the corporate site. 

Something like:

```
        $invoice = $this->invoice;
        $total = $invoice->total();
        $user_name = $this->user->name;
        $vat = $this->user->vat;
        $company_name = $this->user->company;

        $pdf = PDF::loadview('vendor.cashier.receipt', compact('invoice', 'company_name', 'user_name', 'total', 'vat'));

        return (new MailMessage)->markdown('mail.account.invoice-succeeded', [
            'invoice' => $invoice,
            'total'  => $total,
            'user_name'  => $user_name,
            'company_name'  => $company_name,
            'vat' => $vat,
        ])
            ->attachData($pdf->output(), $invoice->id.'-'.date('Y-m-d').'.pdf', ['mime' => 'application/pdf'])
            ->subject('Payment Processed');
```

We obviously don't care about the invoice parts of this, but it can be a helpful guide on how to generate an email with a PDF attachment.